ui/src/logs/containers/LogsPage.tsx

```diff
@@ -16,10 +24,17 @@ interface Props {
   getSources: () => void
   setTimeRange: (timeRange: TimeRange) => void
   setNamespace: (namespace: Namespace) => void
+  changeZoom: (timeRange: TimeRange) => void
+  executeHistogramQuery: () => void
   timeRange: TimeRange
+  histogramData: object[]
 }
 
 class LogsPage extends PureComponent<Props> {
+  constructor(props: Props) {
```

Comment:
do you need this constructor if no state being set?

Reply:
oops, missed this, going to remove.